common/journal_entries/00_sick_man.txt

```diff
@@ -0,0 +1,1007 @@
+		}
+		add_radicals = {
+			culture = cu:bosniak
+			value = 0.15
+		}
+		add_radicals = {
+			culture = cu:bedouin
+			value = 0.15
+		}
+		add_loyalists = {
+			culture = cu:croat
+			value = 0.15
+		}
+		add_loyalists = {
+			culture = cu:sephardic
+			value = 0.15
+		}
+		add_loyalists = {
+			culture = cu:georgian
+			value = 0.15
+		}
+		add_modifier = {
+			name = sick_man_liberal_secularism
+			years = 10
+		}
+		remove_modifier = internal_strife
+	}
+	invalid = {
+		custom_tooltip = {
+			text = finished_sick_man_tt
+			has_variable = sick_man_complete
+		}
+	}
+	transferable = no
+	weight = 5000
+}
+
+# Labor
+je_sick_man_liberal_labor = {
+	icon = "gfx/interface/icons/event_icons/event_industry.dds"
+	complete = {
+		scope:journal_entry = {
+			is_goal_complete = yes
+		}
+		NOR = {
+			has_law = law_type:law_serfdom
+			has_law = law_type:law_tenant_farmers
+		}
+		OR = {
+			has_law = law_type:law_interventionism
+			has_law = law_type:law_laissez_faire
+		}
+		has_law = law_type:law_slavery_banned
+	}
+	on_complete = {
+		set_variable = completed_sick_man_bureaucracy_je
+		custom_tooltip = je_sick_man_advance
+		change_variable = {
+			name = sick_man_var
+			add = 1
+		}
+		add_modifier = {
+			name = sick_man_liberal_labor
+			years = 10
+		}
+	}
+	invalid = {
+		custom_tooltip = {
+			text = finished_sick_man_tt
+			has_variable = sick_man_complete
+		}
+	}
+	transferable = no
+	weight = 5000
+}
+
+# Syria & Egypt
+je_sick_man_syria = {
+	icon = "gfx/interface/icons/event_icons/event_military.dds"
+	complete = {
+		owns_entire_state_region = STATE_ALEPPO
+		owns_entire_state_region = STATE_SYRIA
+		owns_entire_state_region = STATE_LEBANON
+		owns_entire_state_region = STATE_PALESTINE
+		owns_entire_state_region = STATE_TRANSJORDAN
+		owns_entire_state_region = STATE_ADANA
+		OR = {
+			has_law = law_type:law_agrarianism
+			has_law = law_type:law_interventionism
+		}
+	}
+	on_complete = {
+		custom_tooltip = je_sick_man_advance
+		change_variable = {
+			name = sick_man_var
+			add = 1
+		}
+		set_variable = completed_syria_je
+		s:STATE_SINAI = {
+			add_claim = ROOT
+		}
+		s:STATE_UPPER_EGYPT = {
+			add_claim = ROOT
+		}
+		s:STATE_MIDDLE_EGYPT = {
+			add_claim = ROOT
+		}
+		s:STATE_LOWER_EGYPT = {
+			add_claim = ROOT
+		}
+		if = {
+			limit = {
+				OR = {
+					AND = {
+						exists = c:EGY
+						c:EGY = {
+							is_subject_of = ROOT
+						}
+					}
+					AND = {
+						owns_entire_state_region = STATE_SINAI
+						owns_entire_state_region = STATE_UPPER_EGYPT
+						owns_entire_state_region = STATE_MIDDLE_EGYPT
+						owns_entire_state_region = STATE_LOWER_EGYPT
+					}
+				}
+			}
+			custom_tooltip = je_sick_man_advance
+			change_variable = {
+				name = sick_man_var
+				add = 1
+			}
+		}
+		custom_tooltip = sickman_syria_egypt
+	}
+	invalid = {
+		custom_tooltip = {
+			text = finished_sick_man_tt
+			has_variable = sick_man_complete
+		}
+	}
+	on_weekly_pulse = {
+		events = {
+			egyptian_crisis_events.1
```

Comment:
Why is this in weekly_pulse? do you want to fire the event every week?

Reply:
What do you mean? This was Vanilla.

Reply:
well it's silly vanilla. They do a weekly pulse for and event that is only supposed to fire once.

Reply:
idk what you want me to do about that

Reply:
Just checked this out, and it's the event that fires when Ottomans attack Egypt. We _could_ move it to a diplomatic_play on_action instead, and rewrite some of the event triggers?